changelog.d/0-release-notes/pr-2138

```diff
@@ -0,0 +1,2 @@
+Optional team feature config `sndFactorPasswordChallenge` added to galley.yaml.
```

Comment:
I would move this to "internal changes" for now and mark it as "not done yet, please do not enable".  We may have to make a release next week and this would end up in prod and confuse people if they try to turn it on.